### PR TITLE
Reorganizing how solo play sets up. Dropping some reliance on game during set-up.

### PR DESCRIPTION
**Pull request by kberg**

### CODE REVIEW

File: src/Game.ts [hunk 1]
```diff
@@ -231,24 +231,28 @@ export class Game implements ISerializable<SerializedGame> {
 
     const activePlayer = firstPlayer.id;
 
+    // Single player game player starts with 14TR
+    if (players.length === 1) {
```
Comment (nwai90) [inline src/Game.ts:235]: Suggestion: We could replace all instances of `if (players.length === 1)` and `if (this.players.length === 1)` with `if (this.isSoloMode())`

File: src/Game.ts [hunk 2]
```diff
@@ -267,6 +271,12 @@ export class Game implements ISerializable<SerializedGame> {
       game.turmoil = Turmoil.newInstance(game);
     }
 
+    // and 2 neutral cities and forests on board
+    if (players.length === 1) {
```
Comment (nwai90) [inline src/Game.ts:275]: As above

File: src/Game.ts (hunk 1 above)
Comment (kberg) [reply src/Game.ts:235]: NB this occurs prior to Game's construction . I did that on purpose, FWIW.

File: src/Game.ts [hunk 3]
```diff
@@ -231,24 +231,28 @@ export class Game implements ISerializable<SerializedGame> {
 
     const activePlayer = firstPlayer.id;
 
+    // Single player game player starts with 14TR
+    if (players.length === 1) {
+      gameOptions.draftVariant = false;
```
Comment (nwai90) [inline src/Game.ts:236]: This logic could be moved to `GameSetup.initializeSoloGame` or equivalent

**Comment by kberg** [comment]
I like that idea.

>

File: src/Game.ts [hunk 4]
```diff
@@ -1349,6 +1357,12 @@ export class Game implements ISerializable<SerializedGame> {
     });
   }
 
+  public simpleAddTile(player: Player, space: ISpace, tile: ITile) {
```
Comment (Lynesth) [inline src/Game.ts:1360]: Why move that out of the above method ?

**Comment by kberg** [comment]
I'm trying to remove everything that doesn't need a fully-formed game above the actual create - I'd like to find a way to add more common data fully-formed to the constructor, rather than setting them post-construction.

**Review by bafolts — APPROVED** [review]
This seems good enough to merge. There were comments but no one has any requested changes. I will merge this. As the amount of code moving with this project is growing I would like to set standard that if there is something with the pull request that seems enough of an issue to not merge people use the "Request changes" versus only "Comment". I take "Comment" as general comments not worth blocking merging.

**Pull request merged**